File: DailyChallenge/JavaScript/20260107_MarkdownUnorderedListParser.js
function parseUnorderedList(markdown) {
  let list = markdown.split('\n');
  let htmlString = '<ul>';

  for(let i = 0; i < list.length; i++) {
    let listItem = list[i].slice(1).trim();

    htmlString += `<li>${listItem}</li>`;
  }

  htmlString += '</ul>';

  return htmlString;
}

File: Coding Challenges/DailyChallenge/Python/20260107_MarkdownUnorderedListParser.py
def parse_unordered_list(markdown):
    markdown_list = [i[1:].strip() for i in markdown.split('\n')];
    html_string = '<ul>'

    for list_item in markdown_list:
        html_string += f'<li>{list_item}</li>'
    else:
        html_string += '</ul>'

    return html_string